docs/4.2/gcp_guide.md

```diff
@@ -32,7 +32,7 @@ GCP Services required to run Teleport in HA:
 
  - [Compute Engine: VM Instances with Instance Groups](#compute-engine-vm-instances-with-instance-groups)
  - [Computer Engine: Health Checks](#computer-engine-health-checks)
- - [Storage: Cloud Firestore](#storage-cloud-firestore)
+ - [Storage: Cloud Firestore](#storage-google-cloud-storage)
```

Comment:
This link goes to the same place as the one below it. Should there be a separate section?

Reply:
ow, thanks Gus. I've reverted it now.